sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/DefaultErrorResponseException.cs

```diff
@@ -1,62 +0,0 @@
-// <auto-generated>
```

Comment:
Since this change is not functional (meaning this is an orphaned class in the SDK), you should simply not include this change in the PR.